pkg/network/setup/netpod/discoverbridge.go

```diff
@@ -121,14 +121,21 @@ func translateNmstateToNetlinkRoutes(otherRoutes []nmstate.Route) ([]vishnetlink
 // filterRoutesByNonLocalDestination filters out local routes (the destination is of the local link).
 // Default routes should not be filter out.
 func filterRoutesByNonLocalDestination(linkRoutes []nmstate.Route, addr *vishnetlink.Addr) ([]nmstate.Route, error) {
+	_, localNetwork, err := net.ParseCIDR(addr.String())
```

Comment:
Please consider renaming localNetwork such that it express it is a subnet mask
imho it can make it easier to grasp without checking what `ParseCIDR` returns